lib/song_utils/song_normalizer.rb

```diff
@@ -0,0 +1,205 @@
+module SongUtils
+  class BaseParsingStrategy; end
```

Comment:
Awesome use of the strategy pattern here! Let's keep the good coding practice up. Is there a way to split the classes into their own files though? Maybe I'm doing too much Java but I hate scrolling through different classes...